examples/julia-machine-learning/.saturn/saturn.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "example-julia-machine-learning",
```

I think the fact that we're training a neural network is more specific and interesting than general machine learning. So what if we rename the whole thing to reflect that

```suggestion
  "name": "example-julia-neural-network",
```